drivers/SmartThings/matter-window-covering/fingerprints.yml

```diff
@@ -47,3 +47,9 @@ matterGeneric:
     deviceTypes:
       - id: 0x0202 # Window Covering
     deviceProfileName: window-covering
+#WISTAR
+  - id: "5207/18"
+    deviceLabel: WISTAR WSER50 Smart Tubular Motor
```

Comment:
The `deviceLabel` on the cert is "WISTAR Smart Tubular Motor". Where does the WSER50 come from?